site/de/index.html

```diff
@@ -0,0 +1,664 @@
+// [1, 4, 7, 10]
+IntInterval oddsFrom1To10 =
+  IntInterval.oddsFromTo(1, 10); // ungerade ints von 1 bis 10
+// [1, 3, 5, 7, 9]
+IntInterval evensFrom1To10 =
+  IntInterval.evensFromTo(1, 10); // gerade ints von 1 bis 10
+// [2, 4, 6, 8, 10]</pre>
+                        </div>
+                    </div>
+                    <!-- /.tab-content -->
+                </div>
+                <!-- /.col -->
+            </div>
+            <!-- /.row -->
+
+        </div>
+        <!-- /.container -->
+
+    </div>
+    <!-- Container Types -->
+
+    <a class="anchor" name="memory"></a>
+    <div class="content-section-b">
+
+        <div class="container-fluid">
+
+            <div class="row">
+                <div class="col-lg-push-6 col-lg-6 col-sm-push-6 col-sm-6">
+                    <hr class="section-heading-spacer">
+                    <div class="clearfix"></div>
+                    <h2 class="section-heading">Reduziert den Speicherverbrauch<br>mit speichereffizienten Daten-Containern.</h2>
+                    <p class="lead">Eclipse Collections bietet effiziente Implementierungen von Sets und Maps, sowie Primitive Collections. </p>
+                </div>
+                <div class="col-lg-pull-6 col-lg-6 col-sm-pull-6 col-sm-6">
+                    <ul class="nav nav-tabs">
+                        <li class="active"><a href="#set" data-toggle="tab">Sets</a></li>
+                        <li><a href="#map" data-toggle="tab">Maps</a></li>
+                        <li><a href="#ints" data-toggle="tab">Primitive Collections</a></li>
+                    </ul>
+                    <div id="memory" class="tab-content">
+                        <div class="tab-pane fade in active" id="set">
+                            <img class="img-responsive" src="img/set.png" alt="">
+                        </div>
+                        <div class="tab-pane fade" id="map">
+                            <img class="img-responsive" src="img/map.png" alt="">
+                        </div>
+                        <div class="tab-pane fade" id="ints">
+                            <img class="img-responsive" src="img/ints.png" alt="">
+                        </div>
+                    </div>
+                </div>
+            </div>
+
+        </div>
+        <!-- /.container -->
+
+    </div>
+    <!-- Memory footprint -->
+
+    <a class="anchor" name="learn"></a>
+    <div class="content-section-a">
+
+        <div class="container-fluid">
+
+            <div class="row">
+                <div class="col-lg-5 col-sm-6">
+                    <hr class="section-heading-spacer">
+                    <div class="clearfix"></div>
+                    <h2 class="section-heading">Lerne die Eclipse Collections mit Kata</h2>
+                    <p class="lead">
+                        Schau dir das <a href="https://github.com/eclipse/eclipse-collections-kata">Eclipse Collections Kata</a>,
```

Comment:
I would use a hypen instead of the comma here
Eclipse Collections Kata</a>,
=>Eclipse Collections Kata</a> -